Tasks/JavaToolInstallerV0/package.json

```diff
@@ -22,9 +22,14 @@
     },
     "homepage": "https://github.com/microsoft/vsts-tasks#readme",
     "dependencies": {
+        "@types/mocha": "5.2.5",
+        "@types/node": "^6.0.101",
+        "@types/q": "^1.0.7",
         "azure-arm-rest": "file:../../_build/Tasks/Common/azure-arm-rest-1.0.2.tgz",
         "azure-blobstorage-artifactProvider": "file:../../_build/Tasks/Common/azure-blobstorage-artifactProvider-1.0.0.tgz",
+        "mocha": "5.2.0",
```

Comment:
We shouldn't need the dep on mocha.  (Remember that everything here will get packaged and distributed with the task, which has compliance implications.)  @types/mocha can go in a `devDependencies` node under Tests: see https://github.com/Microsoft/vsts-tasks/blob/master/Tasks/CondaEnvironmentV1/Tests/package.json.

We haven't done the work yet, but we plan to strip things declared as dev dependencies out of the packaged task so they aren't distributed.

Reply:
These dependencies were added at the suggestion of Stephen Franceschelli in order to fix a node versioning/dependency issue that surfaced after the update to the vsts-task-tool-lib library. Without these dependencies being declared we were having odd build failures.

Reply:
gotcha

Reply:
We could try to split out the mocha one and see if it still works, I didn't split test each one to make sure we just need node to have it pass(though that would be my guess).